test/unit/test_http_utils.py

```diff
@@ -123,3 +125,19 @@ def server(client, _):
                 self.assertEqual(''.join(rest), data[off:])
         finally:
             server_thread.kill()
+
+
+class TestSomewhatBufferedHTTPConnection(unittest.TestCase):
+
+    def test_discard_buffering_arg_on_py26(self):
+        with mock.patch('httplib.HTTPResponse.__init__') \
+                as http_response_init:
```

Comment:
Prefer prefixing mocks with `mock_` to make clear later in the code that's not a binding to the actual value.